app/components/searchworks4/citations/citation_component.html.erb

```diff
@@ -44,7 +44,7 @@
       <% if eds_ris_export? %>
```

Comment:
We probably want the Refworks + Zotero links to be in separate `<li>` elements so they're arranged horizontally now.